examples/amm/amm.py

```diff
@@ -360,7 +358,6 @@ def burn(
                 pool_bal.hasValue(),
                 a_bal.hasValue(),
                 b_bal.hasValue(),
-                comment=ConstantProductAMMErrors.MissingBalances,
```

Comment:
@barnjamin For my understanding:  In a few places, the PR removes `comment=MissingBalances`.  Can you help me understand the rationale?

Reply:
The comments are useful for conveying a reason to the caller. They also do double duty by flagging an assert as one we'd want to test. In this case, I don't see them as useful to tell the caller about (not to mention tough to test)

Reply:
@barnjamin Got it - works for me.  Resolving.

Out of band to the PR, I think it's worth _some_ consideration to see if we can identify a strategy to more easily test these more difficult-to-test states.